### PR TITLE
Added cifar 100 in examples

### PR DESCRIPTION
**Pull request by abhinavsagar**

### CODE REVIEW

File: examples/cifar100_cnn.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
+'''
+#Train a simple deep CNN on the CIFAR100 images dataset.
+
+It gets to 73% validation accuracy in 25 epochs, and 75% after 50 epochs.
```
Comment (fchollet) [inline examples/cifar100_cnn.py]: The docstring doesn't seem to have  been updated from the CIFAR10 example

File: examples/cifar100_cnn.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+(x_train, y_train), (x_test, y_test) = cifar100.load_data()
+print('x_train shape:', x_train.shape)
+print(x_train.shape[0], 'train samples')
+print(x_test.shape[0], 'test samples')
+
+# Convert class vectors to binary class matrices.
+y_train = keras.utils.to_categorical(y_train, num_classes)
+y_test = keras.utils.to_categorical(y_test, num_classes)
+
+model = Sequential()
```
Comment (fchollet) [inline examples/cifar100_cnn.py]: Please define a state-of-the-art (or relatively close) convnet for CIFAR100, using the functional API, so as to make this example distinct from the CIFAR10 one.

**Comment by abhinavsagar** [comment]
@fchollet Done the necessary changes.

**Comment by abhinavsagar** [comment]
@fchollet Done the necessary changes.

**Comment by abhinavsagar** [comment]
@fchollet Done the necessary changes. I have used VGG with dropout and weight decay as described in the paper.

**Comment by fchollet** [comment]
What test accuracy are you getting after N epochs?

**Comment by fchollet** [comment]
And what paper is this referring to?

**Comment by abhinavsagar** [comment]
> What test accuracy are you getting after N epochs?

After 25 epoch I am getting 73% and after 50 epochs getting 75%.

**Comment by abhinavsagar** [comment]
> And what paper is this referring to?

The corresponding paper is [https://arxiv.org/pdf/1409.1556.pdf](url).

**Comment by abhinavsagar** [comment]
@fchollet Is this ready to be merged or anything further needs to be done ?
